### PR TITLE
ENH: updates from 2/22 beamtime

### PR DESCRIPTION
**Pull request by tangkong**
a few additions to the profile from the February 2022 beamtime.  

Might require some merge conflict managing, but I don't think there's a time constraint to get this sorted out.  

Tested locally.

### CODE REVIEW

**Comment by tangkong** [comment]
This is probably my fault for not managing the `.gitignore` file properly, but we shouldn't sync all the swap files from editing in vim.  

Also we probably don't want to bring all the calibration poni files in.  Maybe a single one is good for example purposes, but all the others will be out of date in future use.

**Comment by tangkong** [comment]
eh we can keep the ponis, might be useful I guess

File: profile_bluesky/startup/instrument/plans/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 from .hitp_scans import *
 from .helpers import *
 from .adapt_opt import *
-from .stats import *
+#from .stats import *
```
Comment (tangkong) [inline profile_bluesky/startup/instrument/plans/__init__.py:4]: Could just delete this line since stats.py doesn't exist anymore

**Comment by tangkong** [comment]
holy merge commits.  I think this is ok now.  @natertott give the changed files a once over to make sure things look like they should, and we can merge

**Review by natertott — APPROVED** [review]
All changes look good to me -- any substantial changes (primarily to survey() and rock()) were made intentionally during the beamtime.

**Pull request merged**